From d0765f7fdce09e3de52988c3dff67bc6e360341b Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Fri, 14 Apr 2023 22:38:34 +0200
Subject: [PATCH 298/406] arm64: dts: rockchip: rk3399-pinephone-pro: Fix
 volume down threshold

It was set too low. My PPP often reports values between 400-500 mV, for
volume down.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index f9ec6177d9d3..5dcff7a72d2b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -48,7 +48,7 @@ button-up {
 		button-down {
 			label = "Volume Down";
 			linux,code = <KEY_VOLUMEDOWN>;
-			press-threshold-microvolt = <300000>;
+			press-threshold-microvolt = <600000>;
 		};
 	};
 
-- 
2.34.1

